<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BugzillaDetails</class>
 <widget class="QWidget" name="BugzillaDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="severityLabel">
     <property name="text">
      <string>Severity</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="priorityLabel">
     <property name="text">
      <string>Priority</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QComboBox" name="severityCombo"/>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="priorityCombo"/>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="statusCombo"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="componentLabel">
     <property name="text">
      <string>Component</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="componentText">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="productLabel">
     <property name="text">
      <string>Product</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="productText">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
